app/schema/user/user.py

```diff
@@ -47,8 +48,9 @@ class User(BaseUser):
     neutral_rates_count: int
     created_at: datetime | str
     is_verified: bool
-    professions: list[Profession]
+    professions: list[Profession] | None
```

Comment:
In most cases its not permitted to set None when we want to get list of some schemas
if professions missing for current job then we want to return EMPTY LIST but NOT NONE